contracts/insured-vesting-v1/InsuredVestingV1.sol

```diff
@@ -6,6 +6,15 @@ import {ProjectRole} from "../roles/ProjectRole.sol";
 import {Address, IERC20, SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
 
+// this contract distributes a project's tokens to users proportionally over a specified period of time, such that tokens are vested
```

Comment:
Perhaps add proper capitlisation here (ie. "This contract...") as you've done on Vesting?